pkg/backend/httpstate/client/client.go

```diff
@@ -472,6 +479,16 @@ func (pc *Client) PublishPolicyPack(ctx context.Context, orgName string,
 		return errors.Wrapf(err, "Failed to upload compressed PolicyPack")
 	}
 
+	//
+	// Step 3: Signal to the service that the PolicyPack publish operation is complete.
+	//
+
+	err = pc.restCall(ctx, "POST",
```

Comment:
nit: you could use `http.MethodPost`

Reply:
Does this restCall take care of situations where we dont get a 200 back?

Reply:
I think it will handle retry on 5xx responses. 4xx responses will be handled by returning an error.

Reply:
I normally do this, but the rest of the client does not. So we do it this way to be consistent.